health/health.d/vcsa.conf

```diff
@@ -0,0 +1,122 @@
+    crit: $this > (($status == $CRITICAL) ? ($update_every) : (60 * $update_every))
+   delay: down 5m multiplier 1.5 max 1h
+    info: number of seconds since the last successful data collection
+      to: sysadmin
+
+# Overall system health:
+#  - 0: all components are healthy.
+#  - 1: one or more components might become overloaded soon.
+#  - 2: one or more components in the appliance might be degraded.
+#  - 3: one or more components might be in an unusable status and the appliance might become unresponsive soon.
+#  - 4: no health data is available.
+
+template: vcsa_system_health
+      on: vcsa.system_health
+  lookup: max -10s unaligned of system
+   units: status
+   every: 10s
+    warn: ($this == 1) || ($this == 2)
+    crit: $this == 3
+   delay: down 1m multiplier 1.5 max 1h
+    info: overall system health status
+      to: sysadmin
+
+# Components health:
+#  - 0: healthy.
+#  - 1: healthy, but may have some problems.
+#  - 2: degraded, and may have serious problems.
+#  - 3: unavailable, or will stop functioning soon.
+#  - 4: no health data is available.
+
+template: vcsa_swap_health
+      on: vcsa.components_health
+  lookup: max -10s unaligned of swap
+   units: status
+   every: 10s
+    warn: $this == 1
+    crit: ($this == 2) || ($this == 3)
+   delay: down 1m multiplier 1.5 max 1h
+    info: swap health status
+      to: sysadmin
+
+template: vcsa_storage_health
+      on: vcsa.components_health
+  lookup: max -10s unaligned of storage
+   units: status
+   every: 10s
+    warn: $this == 1
+    crit: ($this == 2) || ($this == 3)
+   delay: down 1m multiplier 1.5 max 1h
+    info: storage health status
+      to: sysadmin
+
+template: vcsa_mem_health
+      on: vcsa.components_health
+  lookup: max -10s unaligned of mem
+   units: status
+   every: 10s
+    warn: $this == 1
+    crit: ($this == 2) || ($this == 3)
+   delay: down 1m multiplier 1.5 max 1h
+    info: mem health status
+      to: sysadmin
+
+template: vcsa_load_health
+      on: vcsa.components_health
+  lookup: max -10s unaligned of load
+   units: status
+   every: 10s
+    warn: $this == 1
+    crit: ($this == 2) || ($this == 3)
+   delay: down 1m multiplier 1.5 max 1h
+    info: load health status
+      to: sysadmin
+
+template: vcsa_database_storage_health
+      on: vcsa.components_health
+  lookup: max -10s unaligned of database_storage
+   units: status
+   every: 10s
+    warn: $this == 1
+    crit: ($this == 2) || ($this == 3)
+   delay: down 1m multiplier 1.5 max 1h
+    info: database storage health status
+      to: sysadmin
+
+template: vcsa_applmgmt_health
+      on: vcsa.components_health
+  lookup: max -10s unaligned of applmgmt
+   units: status
+   every: 10s
+    warn: $this == 1
+    crit: ($this == 2) || ($this == 3)
+   delay: down 1m multiplier 1.5 max 1h
+    info: appl mgmt health status
+      to: sysadmin
+
+
+# Software updates health:
+#  - 0: no updates available.
```

Comment:
Hi @ilyam8 ,

Overall the alarms are fine, but I did not approve yet due a doubt here, I saw that in this group of comments we are missing the status "- 1", is there a special motive for this?

Reply:
There is no `-1` for software updates according documentation

see https://vmware.github.io/vsphere-automation-sdk-rest/vsphere/index.html#SVC_com.vmware.appliance.health

> Get information on available software updates available in the remote vSphere Update Manager repository.
> Red indicates that security updates are available.
> Orange indicates that non-security updates are available.
> Green indicates that there are no updates available.
> Gray indicates that there was an error retreiving information on software updates.

^^ no yellow (-1) status as you see

Reply:
Hawk eye btw 😄

Reply:
Thank you by the explanation :smile:

Reply:
Just noticed, it is not `-1` it is `1`, i added `-` to show this is a list.